src/modules/helpers/index.ts

```diff
@@ -622,6 +622,29 @@ export class HelpersModule {
     return arrayCopy.slice(min);
   }
 
+  /**
+   * Returns a random value from an Enum object.
+   *
+   * @param enumObject
+   *
+   * @example
+   * faker.datatype.enum(enum Color { Red, Green, Blue}) // 'Green'
+   * faker.datatype.enum(enum Direction { North = 'North', South = 'South'}) // 'South'
+   * faker.datatype.enum(enum HttpStatus { Ok = 200, Created = 201, BadRequest = 400, Unauthorized = 401 }) // 200
+   */
+  enumValue<GenericEnumType extends Object, SpecificEnumType>(
+    enumObject: GenericEnumType
+  ): SpecificEnumType {
+    if(!enumObject) return '0' as SpecificEnumType
+    const ignoreTypeScriptAddedKeys = (enumKeys: string[]) => {
+      return enumKeys.filter((key) => isNaN(Number(key)));
+    };
+    const keys = ignoreTypeScriptAddedKeys(Object.keys(enumObject));
```

Comment:
IMO you can inline this since you only use it once:

```suggestion
    const keys = Object.keys(enumObject)
        // ignore numeric keys added by TypeScript
        .filter((key) => isNaN(Number(key)));
```

(Not sure about the formatting)